core/spec.md

```diff
@@ -2902,21 +2913,69 @@ The following describes the attributes of Registry model:
 
 - `groups."STRING".resources."STRING".attributes`
-  - See [`attributes`](#model.attributes) above and
+  - See [`attributes`](#model.attributes) above,
+    as well as [`resourceattributes`](#model.resourceattributes) and
     [`metaattributes`](#model.metaattributes) below.
   - OPTIONAL.
   - The list of attributes associated with each Version of the Resource.
-  - The list of `groups.resources.attributes` names MUST NOT overlap with the
-    list of `groups.resource.metaattributes` names.
+  - Extension attribute names at this level MUST NOT overlap with extension
+    attributes defined at the `groups.resources.resourceattributes` level.
+    The only duplicate names allowed are specification defined attributes
+    such as `self` and `xid`, and the Version-specific values MUST be
+    overridden by the Resource-level values when serialized.
+
+- `groups."STRING".resources."STRING".resourceattributes`
+  <span id="model.resourceattributes"></span>
+  - See [`attributes`](#model.attributes) above.
+  - OPTIONAL.
+  - The list of attributes associated with the Resource, not its Versions,
+    that will appear in the Resource itself (as siblings to the "default"
+    Version attributes).
+  - These attributes are typically reserved for system-managed attributes
+    such as `metaurl`. User-defined attributes would normally appear on the
+    Versions, and as such will appear in the model under the
+    `groups.resources.attributes` list.
+  - Extension attribute names at this level MUST NOT overlap with extension
+    attributes defined at the `groups.resources.attributes` level.
+    The only duplicate names allowed are specification defined attributes
+    such as `self` and `xid`, and the Version-specific values MUST be
+    overridden by the Resource-level values when serialized.
 
 - `groups."STRING".resources."STRING".metaattributes`
   <span id="model.metaattributes"></span>
   - See [`attributes`](#model.attributes) above.
   - OPTIONAL.
   - The list of attributes associated with the Resource, not its Versions,
-    and will appear in the `meta` sub-object of the Resource.
-  - The list of `groups.resources.attributes` names MUST NOT overlap with the
-    list of `groups.resource.metaattributes` names.
+    that will appear in the `meta` sub-object of the Resource.
+
+Clarifying the  usage of the `attributes`, `resourceattributes` and
+`metaattributes`:
+- The serialization of the Resource is meant to be (almost) interchangeable
+  with the serialization of a Version of that Resource. This will allow users
+  to process either entity without needing to know if they were provided the
+  Resource itself or a specific Version in most cases.
+- To enable this, most of the Resource-specific data (e.g. its
+  `defaultversionid`), is serialized under the `meta` sub-object. This avoids
+  potential name conflicts between Version and Resource-level attributes, as
+  well a avoiding making the serialization of the Resource too verbose/noisy.
```

Comment:
```suggestion
  well as avoiding making the serialization of the Resource too verbose/noisy.
```

I caught your deliberate mistake just to check whether I was reviewing ;)

Reply:
if only I was that smart/sneaky!